dotkom/monoweb
# Feature/dot 191 refund payments

## Checklist


## Changelog

- Adds the possibility to refund payments
- Payments for products may need additional confirmation in order to be refunded
- The correct stripe order id is now set in the database

## How to test

1. Run migrations with `pnpm migrate-with-fixtures`. Make sure all payments are deleted in the DB before doing this.
2. Go to `/payment-test`
3. Click "Fetch products"
4. Copy the stripe command and run it locally
5. Click "Proceed to checkout" and fill in necessary details (Use card 4242 4242 4242 4242)
6. After the payment has gone through, go to `/payment-test/refund`
7. Click "Fetch payments"
8. Click "Refund PaymentId"
9. Click "Fetch refund requests"
10. Approve or reject :)
